boards/nxp/common/Kconfig

```diff
@@ -0,0 +1,10 @@
+# Copyright 2025 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_NXP_SPSDK_IMAGE
+	bool "Generate bootable firmware image with SPSDK"
+	depends on BOARD_IMX95_EVK_MIMX9596_M7
```

Comment:
remove dependency at here as it is 'n' by default, so that it can be enabled by board defconfig if it supports SPSDK image.

Reply:
Removed.